libjcat/jcat-ed25519-engine.c

```diff
@@ -166,28 +253,40 @@ jcat_ed25519_engine_self_verify(JcatEngine *engine,
 				JcatVerifyFlags flags,
 				GError **error)
 {
-	Ed25519Key pubkey = {0};
-	Ed25519Sig sig = {0};
+	g_auto(gnutls_pubkey_t) pubkey = NULL;
```

Comment:
total nit, but normally the `g_auto` stuff goes at the end of the variable block.

Reply:
Fixed.